layouts/_default/baseof.html

```diff
@@ -108,6 +108,10 @@
 
     {{ template "_internal/google_analytics.html" . }}
 
+    {{ if .Site.Params.fathomAnalytics }}
```

Comment:
This template need the ID and the URL to work. So, it is better to also check if the ID was provided. Maybe the URL to if there is no default.

Reply:
Ok, makes sense